maxwellgithinji/Maintenance-Tracker
# #157787979 Implements admin's view of approved request

**What does this PR do?**
Enables the Admin to View the details of an approved user request

**Description of Tasks to be Completed**
_The Admin can:_
*  View the request title of a user.
* View User details
* View the feedback 
* View resolution comment field
*  View user recommendation field
* View the Resolve checkbox
*View a submit button



**How Should This be Manually Tested?**
_After cloning the Repo:_
* Navigate to ```templates/admin_view_approved_detailed.html``` file and open it with a browser.
* Enter some details in the resolution comment field
* Enter some details in the user recommendation field
* Check the Resolve button
* Click on Submit


**Any background context you want to provide?**
The responsiveness on small devices it yet  fully  implemented

**See the Screenshots Below**
![bandicam 2018-05-27 09-41-26-224](https://user-images.githubusercontent.com/11288990/40583212-68efa0c4-6192-11e8-8ff5-0442da95c3d1.jpg)
